package.json

```diff
@@ -80,7 +80,7 @@
     ]
   },
   "peerDependencies": {
-    "eslint": "^6.0.1"
+    "eslint": "^6.2.2"
```

Comment:
If you're using `ncu`, make sure you filter deps. `ncu --dep dev` for example. peerDeps should always be kept on the lowest semver-compatible version that works.

Reply:
up to you